source/class/qxl/apiviewer/ui/panels/InfoPanel.js

```diff
@@ -866,7 +866,7 @@ qx.Class.define("qxl.apiviewer.ui.panels.InfoPanel", {
         itemNode && itemNode.getFullName
           ? itemNode.getFullName()
           : classNode && classNode.getFullName
-          ? classNode.getFullName() + itemName
+          ? `${classNode.getFullName()}${itemName ? itemName: ""}`
```

Comment:
There is an unwanted } in the string

Reply:
Are you sure? There are two expressions and two ${}.

Reply:
sorry you are right